docs/Makefile

```diff
@@ -14,6 +14,13 @@ help:
 
 .PHONY: help Makefile
 
+clean:
+	rm -rf $(BUILDDIR)/*
+	rm -rf examples/
```

Comment:
Can you update `CONTRIBUTING.md` so that it just says to use `make clean` now that clean will remove all temp files?

Reply:
Yes, several changes to CONTRIBUTING.md are necessary.

You've got sharp eyes for this.  I would have glossed over these changes.